types/p5/tslint.json

```diff
@@ -2,7 +2,8 @@
     "extends": "@definitelytyped/dtslint/dt.json",
     "rules": {
         "file-name-casing": false,
-        "unified-signatures": false
+        "unified-signatures": false,
+        "no-unnecessary-class": false
```

Comment:
Please remove this additional disabling of the rule and fix error or disable error on the line where it needs to be

Reply:
fixed